From b7fa0aa00b07e03e338dd02af564431bf2f2b185 Mon Sep 17 00:00:00 2001
From: Joshua Watt <Joshua.Watt@garmin.com>
Date: Wed, 20 Nov 2019 15:24:02 -0600
Subject: [PATCH] Fix source reproducibility

The generated enum type files can be included in source packages meant
for debugging, and thus need to be reproducible. Replace the absolute
include of the header with the basename. This is sufficient because the
target include files are always in the include path anyway.

Upstream-Status: Accepted [https://gitlab.gnome.org/GNOME/at-spi2-core/merge_requests/25]
Signed-off-by: Joshua Watt <Joshua.Watt@garmin.com>
---
 atspi/atspi-enum-types.c.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/atspi/atspi-enum-types.c.template b/atspi/atspi-enum-types.c.template
index 385d0ee..92e4937 100644
--- a/atspi/atspi-enum-types.c.template
+++ b/atspi/atspi-enum-types.c.template
@@ -5,7 +5,7 @@
 
 /*** BEGIN file-production ***/
 /* enumerations from "@basename@" */
-#include "@filename@"
+#include "@basename@"
 
 /*** END file-production ***/
 
-- 
2.23.0

